bson/src/main/org/bson/ByteBuf.java

```diff
@@ -434,7 +434,7 @@ public interface ByteBuf  {
     ByteBuffer asNIO();
 
     /**
-     * Gets the current reference count, which starts at 0.
+     * Gets the current reference count, which starts at 1.
      *
      * @return the current count, which must be greater than or equal to 0
```

Comment:
The "@return ...gte 0" contradicts "starts at 1"?

Reply:
It's not: it's 0 for a released `ByteBuf`.

Reply:
Changed the wording to
> Gets the current reference count, which is 1 for a new {@link ByteBuf}.

Reply:
LGTM (did not review otherwise, just noticed this while using git; so will remove self as reviewer).